### ISSUE TITLE
When a Modal is open keep focus on the Modal

### ISSUE THREAD

**Issue by mtscout6** [react-bootstrap/react-bootstrap#723]
You can currently "tab" off the `Modal` into elements under the `Modal`. This is both bad practice and required for Accessibility.

/cc @jportela

**Comment by citelao** [comment]
The code [here](https://github.com/gdkraus/accessible-modal-dialog/blob/master/modal-window.js#L97) is fairly robust, aside from relying on a constant selector string of tabbable elements (`a[href], area[href], input:not([disabled]), ...`).

Note, the selector string should be `*[tabindex]:not([tabindex=-1])` at the end, instead of `*[tabindex]`.

See [here](https://github.com/hull-ships/hull-login/blob/master/src/components/overlay/index.jsx#L131) for an implementation in React

**Comment by jquense** [comment]
fixed in #810
